errtracker/errtracker.go

```diff
@@ -91,8 +92,13 @@ func snapConfineProfileDigest(suffix string) string {
 	return fmt.Sprintf("%x", md5.Sum(profileText))
 }
 
-func didSnapdReExec() string {
-	if osutil.GetenvBool("SNAP_DID_REEXEC") {
+var didSnapdReExec = func() string {
+	// TODO: move this into osutil.Reexeced() ?
```

Comment:
How about `func osutil.IsReexecedTo(path string) (bool, error)`? Then you could also fix `ExecInCoreSnap()` to use the same osutils helper.

Reply:
it's a bit unclear why it would go into osutil, tbh I see there's already a helper there needing the dirs package but that also seems a bit strange. there's probably some bits of cmd/cmd.go and bits like this that could go make their own helper package.